roles/confluent.variables/defaults/main.yml

```diff
@@ -372,6 +386,9 @@ kafka_rest_ssl_enabled: "{{ssl_enabled}}"
 ### Boolean to enable mTLS Authentication on Rest Proxy
 kafka_rest_ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
 
+### Only use to customize directory Rest Proxy writes log files to. Default location is /var/log/confluent/kafka-rest. NOTE- kafka_rest.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that the Rest Proxy writes log files to. Default location is /var/log/confluent/kafka-rest. NOTE- kafka_rest.appender_log_path is deprecated.